### DISCUSSION TITLE
The Add-On WebUI should have a more comprehensive and condensed view

### DISCUSSION THREAD

**Discussion by CV-Private** [Bepacom-Raalte/bepacom-HA-Addons#45] [Ideas]
I'd wish the WebUI had a more comprehensive and condensed view. Below is a screenshot how the WebUI looks on my screen. For comparison, i put the Yabe window is on top of it. See how much information is visible in Yabe, while the WebUI consists mostly of empty, white space, so that the user has to do a lot of scrolling and clicking.

![image](https://github.com/Bepacom-Raalte/bepacom-HA-Addons/assets/108861703/28c853ed-2c3f-4be4-a6e0-984ab0814ccf)

**Comment by Bepacom-Raalte** [comment]
I agree the web UI is not the best, but to improve it we'd need some help. We're not designers nor web developers, so this is basically the best of our abilities. We welcome anyone to help us of course!

**Comment by GravySeal** [comment]
Made a quick sketch as an idea, would be cool to get something similar to it eventually.

![image](https://github.com/Bepacom-Raalte/bepacom-HA-Addons/assets/93523925/f5306812-dcd6-4541-8b15-38fd839af788)

**Reply by CV-Private** [reply]
The proposed structure looks much better to me! :-)

A few comments (or wishes): 

1) If the mouse pointer hovers over a red button, its color changes to white and it kind of "disappears" on the white background, which can be  quite confusing. I'd suggest that you'd use a different visual effect for "Mouse Over".

2) Static headlines, such as "Devices", are looking exactly the same as Buttons, such as "Write value". I'd suggest that Buttons should be easier to recognize and to differentiate from static UI objects.

3) Please foresee a scroll bar in each column. In the current design, there are scroll bars in drop-down lists, such as Objects, but there's no scroll bar in the Device Browser. If the user has BACnet devices with so many objects as the GTW-21, it becomes painful to navigate without having a scroll bar.

4) Does your WebUI support other color schemes, such as Dark mode? Some users may want to switch themes. I didn't try it yet, but if the Red-White scheme is not changeable, then users with a preference for Dark mode may not be happy...

5) In your new WebUI, please mention "BACnet/IP Interface" in the header. It is on the current WebUI, but missing in your sketch.
